server/repository/src/migrations/v2_03_00/add_reverse_prescription_inventory_addition_reason.rs

```diff
@@ -0,0 +1,22 @@
+use crate::migrations::*;
+use util::constants::REVERSE_PRESCRIPTION_REASON_ID;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_reverse_prescription_inventory_addition_reason"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+               INSERT INTO inventory_adjustment_reason (id, type, is_active, reason) 
+                VALUES ('{REVERSE_PRESCRIPTION_REASON_ID}', 'POSITIVE', TRUE, '{REVERSE_PRESCRIPTION_REASON_ID}');
+            "#
+        )?;
+
+        Ok(())
```

Comment:
Just wondering, if this is a "system" reason that is always available, perhaps it would be better to hard-code it rather than put it in the database?

Putting it in the database makes it susceptible to user tampering. I know we can prevent it being edited, but if it wasn't here, then we wouldn't need the additional filtering/handling you've added, and we could just inject into the list of reasons (in the service layer?) in the few places it's needed?